awswrangler/distributed/ray/_register.py

```diff
@@ -0,0 +1,44 @@
+"""Ray and Modin registered methods (PRIVATE)."""
+# pylint: disable=import-outside-toplevel
+from awswrangler._distributed import EngineEnum, MemoryFormatEnum, memory_format
+from awswrangler._utils import table_refs_to_df
+from awswrangler.distributed.ray._utils import _batch_paths_distributed
+from awswrangler.s3._read_parquet import _read_parquet
+from awswrangler.s3._read_text import _read_text
+from awswrangler.s3._wait import _batch_paths
+from awswrangler.s3._write_dataset import _to_buckets, _to_partitions
+from awswrangler.s3._write_parquet import _to_parquet
+from awswrangler.s3._write_text import _to_text
+
+
+def register_ray() -> None:
+    """Register dispatched Ray and Modin (on Ray) methods."""
+    execution_engine = EngineEnum.RAY.value
+
+    _batch_paths.register(execution_engine, _batch_paths_distributed)  # type: ignore
+
+    if memory_format.get() == MemoryFormatEnum.MODIN.value:
```

Comment:
Really cool, I love how this is out of way!

Reply:
+1. Minor Q - can `memory_format.get()` return an enum so you don't have to do `.value` on every comparison?